### ISSUE TITLE
`aria-brailleroledescription` page title is not formatted correctly.

### ISSUE THREAD

**Issue by cmdNiels** [mdn/content#23091]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-brailleroledescription

### What specific section or headline is this issue about?

Sidebar Heading

### The title of the page is not formatted correctly.

Currently Is: `[**WAI-ARIA states and properties**](/en-US/docs/Web/Accessibility/ARIA/Attributes)`

### The title of the page is not formatted correctly.

Should be: `WAI-ARIA states and properties`

### Do you have anything more you want to share?

![image](https://user-images.githubusercontent.com/73907007/208650700-d05f4bec-1546-4fc8-945a-c0e9253d825d.png)
